### PR TITLE
Welcome log

### PR DESCRIPTION
**Pull request by Lyst67**
Пофіксив помилки.

### CODE REVIEW

**Comment by Sokilskill** [comment]
Працює реєстрація, логін?

**Comment by Lyst67** [comment]
Ні.

16 січня 2024, 18:21:57, від "Serhii" ***@***.***>:


Працює реєстрація, логін?
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>

--=-WcBIO9JaK9QPyWbWfUdG
Content-Type: text/html; charset="UTF-8"
Content-Transfer-Encoding: base64

PGh0bWw+PGJvZHk+PHNwYW4gc3R5bGU9ImRpc3BsYXk6YmxvY2s7IiBjbGFzcz0ieGZtXzc5NzYz
NTQ1Ij48ZGl2PjxzcGFuIHN0eWxlPSJmb250LXNpemU6MTJwdDtsaW5lLWhlaWdodDoxNHB0O2Zv
bnQtZmFtaWx5OkFyaWFsOyIgY2xhc3M9InhmbWMxIj7QndGWLjwvc3Bhbj48YnIvPjwvZGl2Pg0K
PGRpdj48YnIvPjwvZGl2Pg0KPGRpdj48aT48c3BhbiBzdHlsZT0iZm9udC1zaXplOjEwcHQ7bGlu
ZS1oZWlnaHQ6MTJwdDsiPjxzcGFuIHN0eWxlPSJmb250LWZhbWlseTpBcmlhbDsiPjE2INGB0ZbR
h9C90Y8gMjAyNCwgMTg6MjE6NTcsINCy0ZbQtA0KICAgICJTZXJoaWkiICZsdDs8L3NwYW4+PGEg
aHJlZj0ibWFpbHRvOm5vdGlmaWNhdGlvbnNAZ2l0aHViLmNvbSIgdGFyZ2V0PSJfYmxhbmsiPjxz
cGFuIHN0eWxlPSJmb250LWZhbWlseTpBcmlhbDsiPm5vdGlmaWNhdGlvbnNAZ2l0aHViLmNvbTwv
c3Bhbj48L2E+PHNwYW4gc3R5bGU9ImZvbnQtZmFtaWx5OkFyaWFsOyI+Jmd0OzoNCjwvc3Bhbj48
L3NwYW4+PC9pPjwvZGl2Pg0KPGRpdj48YnIvPjwvZGl2Pg0KPGJsb2NrcXVvdGUgc3R5bGU9ImJv
cmRlci1sZWZ0OjFweCBzb2xpZCAjY2NjY2NjO21hcmdpbjowcHggMHB4IDBweCAwLjhleDtwYWRk
aW5nLWxlZnQ6MWV4OyI+DQo8ZGl2IHN0eWxlPSJkaXNwbGF5OmJsb2NrOyI+DQo8ZGl2PjwvZGl2
Pg0KPGRpdj7Qn9GA0LDRhtGO0ZQg0YDQtdGU0YHRgtGA0LDRhtGW0Y8sINC70L7Qs9GW0L0/PC9k
aXY+DQo8ZGl2PjxzcGFuIHN0eWxlPSJmb250LXNpemU6c21hbGw7Y29sb3I6IzY2NjsiPuKAlDxi
ci8+UmVwbHkgdG8gdGhpcyBlbWFpbCBkaXJlY3RseSwgPGEgaHJlZj0iaHR0cHM6Ly9naXRodWIu
Y29tL1Nva2lsc2tpbGwvcHJvamVjdC1kYWlseS1lbmVyZ3ktZG9zZS9wdWxsLzIxI2lzc3VlY29t
bWVudC0xODk0MDc3MDQzIiB0YXJnZXQ9Il9ibGFuayIgcmVsPSJub3JlZmVycmVyIG5vb3BlbmVy
Ij52aWV3IGl0IG9uIEdpdEh1YjwvYT4sIG9yIDxhIGhyZWY9Imh0dHBzOi8vZ2l0aHViLmNvbS9u
b3RpZmljYXRpb25zL3Vuc3Vic2NyaWJlLWF1dGgvQU8zNVVLMk5YNjRGRjJSQklSR0VINzNZTzJT
S0pBVkNORlNNNkFBQUFBQkI1Qk5QSUNWSEkyRFNNVlFXSVgzTE1WNDNPU0xUT04yV0tRM1BOVldX
SzNUVUhNWVRRT0pVR0EzVE9NQlVHTSIgdGFyZ2V0PSJfYmxhbmsiIHJlbD0ibm9yZWZlcnJlciBu
b29wZW5lciI+dW5zdWJzY3JpYmU8L2E+Ljxici8+WW91IGFyZSByZWNlaXZpbmcgdGhpcyBiZWNh
dXNlIHlvdSBhdXRob3JlZCB0aGUgdGhyZWFkLjxpbWcgc3JjPSJodHRwczovL2dpdGh1Yi5jb20v
bm90aWZpY2F0aW9ucy9iZWFjb24vQU8zNVVLWVBPTUpDUElEM0NOQ0E3WDNZTzJTS0pBNUNORlNN
NkFBQUFBQkI1Qk5QSUNXR0czM05OVlNXNDVDN09SNFhBWk5NSkZaWEc1TEZJTlhXMjNMRk5aMktV
WTNQTlZXV0szVFVMNVVXSlRUUTRWSkhHLmdpZiIgaGVpZ2h0PSIxIiB3aWR0aD0iMSIgYWx0PSIi
Lz48L3NwYW4+PHNwYW4gc3R5bGU9ImZvbnQtc2l6ZTowO2NvbG9yOnRyYW5zcGFyZW50O2Rpc3Bs
YXk6bm9uZTtvdmVyZmxvdzpoaWRkZW47d2lkdGg6MDtoZWlnaHQ6MDttYXgtd2lkdGg6MDttYXgt
aGVpZ2h0OjA7Ij5NZXNzYWdlIElEOiAmbHQ7U29raWxza2lsbC9wcm9qZWN0LWRhaWx5LWVuZXJn
eS1kb3NlL3B1bGwvMjEvYzE4OTQwNzcwNDNAZ2l0aHViLmNvbSZndDs8L3NwYW4+PC9kaXY+DQo8
L2Rpdj4NCjwvYmxvY2txdW90ZT48L3NwYW4+PC9ib2R5PjwvaHRtbD4NCg==

--=-WcBIO9JaK9QPyWbWfUdG--

File: src/redux/auth/auth-operations.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { createAsyncThunk } from '@reduxjs/toolkit';
 import axios from 'axios';
 
-axios.defaults.baseURL = 'https://power-pulse-6-backend.onrender.com/api-docs/';
+axios.defaults.baseURL = 'https://power-pulse-6-backend.onrender.com';
```
Comment (Sokilskill) [inline src/redux/auth/auth-operations.js:4]: https://power-pulse-6-backend.onrender.com/api

**Review by Sokilskill — CHANGES_REQUESTED** [review]
Зміни base url
z-index закоментований

**Review by Sokilskill — APPROVED** [review]

**Pull request merged**